### PR TITLE
Prepare for insert api

### PR DESCRIPTION
**Pull request by yhmo**
Signed-off-by: yhmo <yihua.mo@zilliz.com>

### CODE REVIEW

File: src/include/types/FieldData.h [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+using BoolFieldData = FieldData<bool, DataType::BOOL>;
+using Int8FieldData = FieldData<int8_t, DataType::INT8>;
+using Int16FieldData = FieldData<int16_t, DataType::INT16>;
+using Int32FieldData = FieldData<int32_t, DataType::INT32>;
+using Int64FieldData = FieldData<int64_t, DataType::INT64>;
+using FloatFieldData = FieldData<float, DataType::FLOAT>;
+using DoubleFieldData = FieldData<double, DataType::DOUBLE>;
+using BinaryVecFieldData = FieldData<std::vector<uint8_t>, DataType::BINARY_VECTOR>;
+using FloatVecFieldData = FieldData<std::vector<float>, DataType::FLOAT_VECTOR>;
+
```
Comment (matrixji) [inline src/include/types/FieldData.h:161]: Do we need StringFieldData for DataType::STRING?

Comment (yhmo) [reply src/include/types/FieldData.h:161]: Right. I missed.

**Pull request merged**